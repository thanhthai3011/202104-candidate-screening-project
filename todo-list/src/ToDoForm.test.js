import renderer from 'react-test-renderer';
import ToDoForm from './ToDoForm';


test('', () => {
  const component = renderer.create(
    <ToDoForm />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // manually trigger the callback
  tree.props.handleChange();
  // re-rendering
  tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // manually trigger the callback
  tree.props.handleSubmit();
  // re-rendering
  tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
