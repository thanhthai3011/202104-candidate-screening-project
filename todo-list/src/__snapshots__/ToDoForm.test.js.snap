// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    placeholder="Enter task..."
    type="text"
    value=""
  />
  <button>
    Submit
  </button>
</form>
`;

exports[`To Do List 1`] = `
<header>
  <h1>
    To Do List
  </h1>
</header>
`;
